zeppelin-server/src/main/java/org/apache/zeppelin/rest/NotebookRestApi.java

```diff
@@ -1147,12 +1147,16 @@ public Response getCronJob(@PathParam("noteId") String noteId)
   @ZeppelinApi
   public Response getJobListforNote() throws IOException, IllegalArgumentException {
     LOGGER.info("Get note jobs for job manager");
-    List<JobManagerService.NoteJobInfo> noteJobs = jobManagerService
-            .getNoteJobInfoByUnixTime(0, getServiceContext(), new RestServiceCallback<>());
-    Map<String, Object> response = new HashMap<>();
-    response.put("lastResponseUnixTime", System.currentTimeMillis());
-    response.put("jobs", noteJobs);
-    return new JsonResponse<>(Status.OK, response).build();
+    try {
```

Comment:
The try-catch style is not used in this class. Please change the style of the implementation to the following structure.

```
- checkjobManagerEnabled()  --> this method can throw an RestException
- JobManager.anything()
```